hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/POSTGRESQL/V1_12_24__add_confirmation_flag___POSTGRESQL.sql

```diff
@@ -0,0 +1,34 @@
+ALTER TABLE sp_rolloutgroup
+    ADD COLUMN confirmation_required BOOLEAN;
+ALTER TABLE sp_target_filter_query
+    ADD COLUMN confirmation_required BOOLEAN;
+
+UPDATE sp_base_software_module SET confirmation_required = 0;
+UPDATE sp_target_filter_query SET confirmation_required = 0;
+
+CREATE TABLE sp_target_conf_status
+(
+    id               BIGINT      NOT NULL DEFAULT nextval('sp_target_conf_status_seq'),
```

Comment:
Please replace `id               BIGSERIAL   NOT NULL DEFAULT nextval('sp_target_conf_status_seq'),` by `id               BIGSERIAL`. According to https://www.postgresql.org/docs/current/datatype-numeric.html#DATATYPE-SERIAL it will be the same:
```
CREATE TABLE tablename (
    colname SERIAL
);
```
is equivalent to specifying:
```
CREATE SEQUENCE tablename_colname_seq AS integer;
CREATE TABLE tablename (
    colname integer NOT NULL DEFAULT nextval('tablename_colname_seq')
);
ALTER SEQUENCE tablename_colname_seq OWNED BY tablename.colname;
```